pom.xml

```diff
@@ -183,6 +190,17 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
+                <version>1.15.10</version>
```

Comment:
shouldn't you add scope to these?

Reply:
@wikumChamith did you see the above comment?

Reply:
Yeah. I'll update this

Reply:
@dkayiwa I updated the PR.